CONTRIBUTING.md

```diff
@@ -136,6 +136,51 @@ The last line of commits introducing breaking changes should be in the form `BRE
 
 Breaking changes should also add an exclamation mark `!` after the type/scope (e.g. `refactor(rt)!: drop support for Android API < 20`)
 
+### Automated PR checks
+
+A number of automated workflows run when a PR is submitted. Generally speaking, each of these must pass before the PR is
+allowed to be merged. If your PR fails one of the checks, please attempt to address the problem and push a new commit to
+the PR. If you need help understanding or resolving a PR check failure, please reach out via a PR comment, a GitHub
+discussion, or a new issue.
```

Comment:
We might want to clarify to only open a new issue if they believe / confirm with us they found an existing, separate issue. I'm not sure we want issues opened to request help fixing PRs.